tests/unit/v1/test_watch.py

```diff
@@ -850,6 +870,9 @@ class DummyFirestore(object):
     _database_string = "abc://bar/"
     _rpc_metadata = None
 
+    def ListenRequest(self, **kw):  # pragma: NO COVER
+        pass
+
```

Comment:
Confused about the purpose of this? Not sure what `pragma: NO COVER` means.

Reply:
This instructs the coverage to not try and cover this specific line of code.